apps/web/lib/QueryCell.tsx

```diff
@@ -88,7 +88,7 @@ const withQuery = <
   TInput = inferProcedureInput<TQuery>,
   TOutput = inferProcedureOutput<TQuery>
 >(
-  queryProcedure: DecorateProcedure<TQuery, any, any>,
```

Comment:
fixes this error while building

![Screenshot 2023-02-14 14:12:07](https://user-images.githubusercontent.com/84864519/218684382-8ed85be6-767f-4a99-b26b-9d4e6827df04.png)